bots/IssueCommands.txt

```diff
@@ -34,6 +34,11 @@ close
 comment {author} has closed this issue. If you think it should remain open, let us know why. See ["What to Expect from Maintainers"](https://facebook.github.io/react-native/docs/maintainers.html#handling-issues).
 close
 
+@facebook-github-bot large-pr
+comment Thank you for your contribution. As it happens, the pull request size seems relatively large. In order to reduce the load on maintainers, it would be valuable if you could split this into smaller PRs that were very targeted that changed one thing at a time. It could be valuable specifying the goal and steps your PRs will take, in a new issue if one does not already exist, in order to ensure maintainers have context across the stack of PRs. We have added the tag large-pr and closed this task. If Pull Request contains multiple changes, split each into separate PR will helps faster, easier review. If this is a codemod or other formatting change that is simple but inheritly touches many files, please comment on this and let us know and we will reopen.
```

Comment:
It may also be worth linking to something about how and why this is valuable.

A quick google search brought up a post that @breerly wrote that looks good. https://graysonkoonce.com/stacked-pull-requests-keeping-github-diffs-small/